code/game/machinery/portable_turret.dm

```diff
@@ -614,21 +616,36 @@ GLOBAL_LIST_EMPTY(turret_icons)
 		if(target(M))
 			return 1
 
-/obj/machinery/porta_turret/proc/popUp()	//pops the turret up
+/obj/machinery/porta_turret/proc/check_popUp()
```

Comment:
Also if you renamed `popUp()` to `pop_up()` why not make this `check_pop_up()`

Reply:
good pont well made